source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,110 @@
+
+## Installing Minishift
+
+- Download the archive for your OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it.
+- Copy the contents of the directory to your preferred location.
+- Add the minishift binary to your PATH environment variable.
+
+Other alternative ways to install Minishift can be found [here](https://github.com/Minishift/minishift#installing-minishift).
+
+## Quickstart with Minishift
+
+- Get start as follows:
+
+    ```
+    $ minishift start
+     Starting local OpenShift cluster using 'kvm' hypervisor...
+     ...
+        OpenShift server started.
+        The server is accessible via web console at:
+            https://192.168.99.128:8443
+
+        You are logged in as:
+            User:     developer
+            Password: developer
+
+        To login as administrator:
+            oc login -u system:admin
+    ```
+
+    This will download the latest [_minishift-b2d-iso_](https://github.com/Minishift/minishift-b2d-iso), starts a VM using libmachine, download OpenShift client binary (`oc`) with version `1.4.1` and cache both `oc` and ISO into your `$HOME/.minishift/cache` folder and provisions OpenShift single node cluster.
+
+    This step also automatically logged you in with `developer` user in `myproject` project.
+
+- Export the OpenShift client binary (`oc`) to _PATH_ environment variable as follows:
+
+    ```
+    $ export PATH=$PATH:$HOME/.minishift/cache/oc/v1.4.1
+    ```
+
+- Deploy a sample application as follows:
+
+    - Create a Nodejs example application:
+
+         ```
+         $ oc new-app https://github.com/openshift/nodejs-ex -l name=myapp
+         ```
+    - Track the build log until the app is built and deployed using:
+
+        ```
+        $ oc logs -f bc/nodejs-ex
+        ```
+    - Expose a route to the service as follows:
+
+        ```
+        $ oc expose svc/nodejs-ex
+        ```
+    - Open the application in browser as follows:
+
+        ```
+        $ minishift openshift service nodejs-ex -n myproject
+        ```
+
+- When done with building application for the day, stop the cluster as follows:
+
+    ```
+     $ minishift stop
+     Stopping local OpenShift cluster...
+     Stopping "minishift"...
+    ```
+
+    Now, on some another day, if you want to access this cluster, just use `minishift start`.
+
+- At anytime if you want to delete the cluster, use the following command:
+
+    ```
+    $ minishift delete
+    ```
+
+[Minishift's  README](https://github.com/Minishift/minishift#minishift) has more detailed description of getting start with single node OpenShift cluster in different OS platforms.
+
+## Feedback
+
+Try Minishift! We would love to get your feedback. If you hit by any problem, you can open the issue in [minishift issue tracker](https://github.com/minishift/minishift/issues) but please search through it before creating it. It's possible that the similar issue is already open.
```

Comment:
If you hit a problem, please raise an issue...
through the listed issues
a similar